portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,10 +27,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add("I crochet too!");
+    comments.add("I'm actually majoring in underwater basket weaving.");
+    comments.add("I garden as well but only cacti.");
+    comments.add("My secret talent is that I can whistle all of Queen's songs.");
+    comments.add("I'm a champion cup-stacker (I won gold in the AAU Junior Olympics). ");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String title = "I am Isabel Mitre";
-    response.setContentType("text/html;");
-    response.getWriter().println(title);
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
```

Comment:
Similar to my other comment, you may want to check that `comments` is initialized and non-empty. In this case, if `toJson` can accept null, it's also ok to leave a comment explaining what the expected behavior is.

Reply:
I made a separate file for the user inputted comments so can I just leave this one as is?

Reply:
Yeah that sounds fine